poetry.lock

```diff
@@ -242,6 +242,14 @@ djangorestframework = ">=3.9"
 python = "<3.8"
 version = ">=3.7.4"
 
+[[package]]
+category = "main"
+description = "Guppy 3 -- Guppy-PE ported to Python 3"
+name = "guppy3"
+optional = false
+python-versions = ">=3.5"
+version = "3.0.10.post1"
+
```

Comment:
Do you use guppy3 actually?

Reply:
I used it to debug the heap, but you're right, it doesn't need to stay permanently in the project. Going to remove.